src/inspect.rs

```diff
@@ -8,20 +10,38 @@ use pulldown_cmark::{Options, Parser};
 use std::convert::TryFrom;
 use syntect::parsing::SyntaxSet;
 
-pub(crate) fn inspect(uri: &str, output: OutputType) -> Result<()> {
+pub(crate) async fn inspect(
+    uri: &str,
+    output: OutputType,
+    sources: Option<Sources>,
+    docker_config: Option<DockerConfig>,
+) -> Result<()> {
     let uri = crate::utils::map_path_to_uri(uri)?;
     let wasm_path = crate::utils::wasm_path(uri.as_str())?;
-    let printer = get_printer(output);
+    let printer = get_printer(&output);
 
     let metadata = Metadata::from_path(&wasm_path)
         .map_err(|e| anyhow!("Error parsing policy metadata: {}", e))?;
+
+    let signatures = fetch_signatures_manifest(uri.as_str(), sources, docker_config).await;
```

Comment:
I would move this line below the `match metadata` block.

Reply:
If I move this line below the match, there will be a delay when displaying the signatures.
It will first print the metadata, then do a http request to fetch the signatures and display them. It feels a bit strange for the user to see the metada printed and after maybe one or two seconds (probably less depending on the network) the signatures. What do you think?

Reply:
Makes sense, thanks for the explanation